### ISSUE TITLE
Attach to nginx?

### ISSUE THREAD

**Issue by vanheck** [tomblind/local-lua-debugger-vscode#1]
Hello, is it possible to attach debuger to nginx lua script?

**Comment by tomblind** [comment]
I'm not familiar with nginx or it's lua support, so I can't say for sure if it's supported.

It might work if you can launch nginx as a custom lua executable and if it supports lua's `io.read` and `io.write` functions. But, if you're trying to attach to a running nginx instance, it won't work.

**Comment by vanheck** [comment]
ZeroBrane Studio works fine for remote debuging with nginx and any other lua extensions. https://studio.zerobrane.com/doc-remote-debugging - they using MobDebug https://github.com/pkulchenko/MobDebug (I'm not sure if it's the right repo). I not find the solution which works with VS Code.

**Comment by tomblind** [comment]
Yes, remote debugging is the big missing feature of this plugin right now. There are other VSCode extensions that support it, though. I believe nginx uses luajit, right? Maybe one of these will work:
- https://marketplace.visualstudio.com/items?itemName=devCAT.lua-debug
- https://marketplace.visualstudio.com/items?itemName=beamng.luadbg

At some point, when I get time, I may add support for remote debugging.

**Comment by vanheck** [comment]
Adding remote debuging feature will be fine, I like your vscode extension, it works well under Linux (I tested linux only). 

Yes, nginx using luajit. These extension, especially [the first one](https://marketplace.visualstudio.com/items?itemName=devCAT.lua-debug) doesn't work in linux and support and documentation is in chinesse (my guess) only :/.
